### ISSUE TITLE
Test to verify replicator establishes new connection when drive comes back online

### ISSUE THREAD

**Issue by pauldardeau** [swiftstack/kinetic-swift#52]
We need a test to verify that the replicator establishes a new connection when a drive comes back online after being temporarily unavailable.  We believe that the replicator may not currently correctly re-establish a connection.

**Comment by clayg** [comment]
Yeah I did some manual testing when I was finishing up 70160edf - but I don't see a test for it - so I'm not sure if there'd be a good way to do a unit test for that?

Maybe copy test_replicator.TestUtilFunctions.test_ec_handoff - cause it calls stop_simulator

**Comment by clayg** [comment]
I still don't have a real good handle on this one.  In development I can shut down devices/simulators and everything reconnects failure well.

But in testing one time I saw an traceback where the P2P commands would cause the client to spew a  a "connection forcibly closed by peer" that caused the client to think it's connection was faulted and it would get stuck.

I wasn't able to grab the error message at the time, and now looking back at the connection pooling in the replicator it looks like it should be checking for a faulted connection and reconnecting automatically.

Maybe I was running an older version, I changed the default deployment option to COPY and so far haven't seen the issue again.
